src/components-examples/cdk/stepper/cdk-linear-stepper-with-form/example-custom-linear-stepper.html

```diff
@@ -0,0 +1,22 @@
+<section class="example-container">
+    <header>
+        <h2>Step {{ selectedIndex + 1}}/{{ steps.length}}</h2>
+    </header>
+
+    <div [style.display]="selected ? 'block' : 'none'">
+        <ng-container [ngTemplateOutlet]="selected.content"></ng-container>
+    </div>
+
+    <footer class="example-step-navigation-bar">
+        <button class="example-nav-button" cdkStepperPrevious>&larr;</button>
+        <button
+                class="example-step"
+                *ngFor="let step of steps; let i = index"
+                [ngClass]="{ 'example-active': selectedIndex === i }"
```

Comment:
If you only have one class, it's usually better to use a single `class` binding: `[class.example-active]="selectedIndex === i"`.

Reply:
Using only [class.example-active]="selectedIndex === i" does not give the desired style, but I adjusted it to be similar to your suggesting with only one class binding. I believe it now works. Thanks.